iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -22,7 +22,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.17.0-preview-006</Version>
+    <Version>1.17.0-preview-007</Version>
```

Comment:
Are these the only places we need to change? Also, can you please confirm that nothing has changed in Shared/Service SDKs, say maybe because of the FIs?

Reply:
only device sdk changes